src/action.ts

```diff
@@ -1,9 +1,11 @@
 import * as core from '@actions/core'
 import {check_directory, check_urls} from './check'
+import {set_up_configuration} from './config'
 
 export async function main(): Promise<void> {
     try {
         const repositories: string = core.getInput('repositories')
```

Comment:
what about other inputs like curations, rules, classifications?
I think you should also use `getInput` for these variables.